src/leaderboard/messages/data/get_competition_rank.rs

```diff
@@ -10,12 +10,24 @@ pub struct GetCompetitionRankRequest {
 }
 
 impl GetCompetitionRankRequest {
-    pub fn new(ids: impl IntoIterator<Item = u32>, order: Option<Order>) -> Self {
+    /// Constructs a new `GetCompetitionRankRequest`.
+    ///
+    /// Defaults to DESCENDING order, meaning that rank 0
+    /// is the element with the highest score.
+    pub fn new(ids: impl IntoIterator<Item = u32>) -> Self {
         Self {
             ids: ids.into_iter().collect(),
-            order: order.unwrap_or(Order::Descending),
+            order: Order::Descending,
         }
     }
+
+    /// Sets the order ranking.
+    ///
+    /// Defaults to DESCENDING order.
+    pub fn order(mut self, order: Order) -> Self {
+        self.order = order;
+        self
+    }
 }
```

Comment:
Should default to ascending following `get_rank` and the various fetch methods, eg in JS [see](https://github.com/momentohq/client-sdk-javascript/blob/main/packages/core/src/internal/clients/leaderboard/AbstractLeaderboard.ts#L120-L132)

Reply:
Yea there was a discussion about this in the [PRD](https://www.notion.so/momentohq/PRD-Leaderboards-Competition-Ranking-17afeb15c97d8019aa15d508a8188083).
Our posture here is that for competition ranking so we should default to descending.

Reply:
Got it!